src/main/scala/cromwell/engine/backend/jes/JesEngineFunctions.scala

```diff
@@ -9,30 +9,36 @@ import cromwell.util.google.GoogleCloudStoragePath
 import scala.language.postfixOps
 import scala.util.{Success, Try}
 
+object JesEngineFunctions {
+  implicit class UriString(val str: String) extends AnyVal {
+    def isGcsUrl: Boolean = str.startsWith("gs://")
```

Comment:
I lost track - was this originally in the code block that Chris was talking about earlier?